### DISCUSSION TITLE
S14-24V3 Clock Can you provide the pcb circuit diagram?

### DISCUSSION THREAD

**Discussion by zx804262156** [simpleavr/s14clock#15] [Q&A] [answered]
The one I made doesn't work properly

**Comment by simpleavr** [comment]
Do you have V2 or V3 PCB?

**Comment by simpleavr** [comment, marked as answer]
[Schematic_S14-24V3.pdf](https://github.com/simpleavr/s14clock/files/12233110/Schematic_S14-24V3.pdf)
[Schematic_S14-24V2.pdf](https://github.com/simpleavr/s14clock/files/12233111/Schematic_S14-24V2.pdf)

Attached are both version schematics.
